docs/src/main/sphinx/connector/ignite.rst

```diff
@@ -0,0 +1,156 @@
+----------------
+
+Table property usage example::
+
+    CREATE TABLE public.person (
+      id bigint NOT NULL,
+      birthday DATE NOT NULL,
+      name VARCHAR(26),
+      age BIGINT,
+      logdate DATE
+    )
+    WITH (
+      primary_key = ARRAY['id', 'birthday'],
+      affinity_key = 'birthday',
+      template = 'REPLICATED',
+      write_synchronization_mode = 'FULL_ASYNC'
+    );
+
+The following are supported Ignite table properties from `<https://https://ignite.apache.org/docs/latest/sql-reference/ddl>`_
+
+=============================== ==============================================================================================================
+Property Name                   Description
+=============================== ==============================================================================================================
+``primary_key``                 The primary key of the table, can chose multi columns as the table primary key. Table at least contains one
+                                column not in primary key. Required.
+
+``affinity_key``                The affinity_key of the table. Must be one of the ``primary_key``. Optional.
+
+``template``                    The storage template about the table. Available values are ``REPLICATED`` and ``PARTITIONED`. Default value is
+                                ``PARTITIONED``. Optional.
+
+``BACKUPS``                     The replication numbers for the table, must greater than 0. Default value is 1. Optional.
+
+``cache_group``                 Specifies the group name the underlying cache belongs to. Default value is prefix ``SQL_PUBLIC_`` append with
+                                UPPERCASE of the table name. Optional.
+
+``cache_name``                  The name of the underlying cache created by the command. Default value is prefix ``SQL_PUBLIC_`` append with
+                                UPPERCASE of the table name. Optional.
+
+``write_synchronization_mode``  Sets the write synchronization mode for the underlying cache. If neither this nor the ``TEMPLATE`` parameter
+                                is set, then the cache is created with ``FULL_SYNC`` mode enabled. Available values are ``PRIMARY_SYNC``,
+                                ``FULL_SYNC`` and ``FULL_ASYNC``. Optional.
+
+``data_region``                 ame of the data region where table entries should be stored. By default, Ignite stores all the data in a
+                                default region. Optional.
+=============================== ==============================================================================================================
+
+Pushdown
+--------
+
+The connector supports pushdown for a number of operations:
+
+* :ref:`join-pushdown`
+* :ref:`limit-pushdown`
+* :ref:`topn-pushdown`
+
+:ref:`Aggregate pushdown <aggregation-pushdown>` for the following functions:
+
+* :func:`avg`
+* :func:`count`
+* :func:`max`
+* :func:`min`
+* :func:`sum`
+
+Limitations
```

Comment:
Can you please add how Ignite types are mapped to Trino types and vice versa?

Reply:
We are standardizing on a new approach... ideally you could update to that (see sqlserver.rst and postgresql.rst in master branch)